generators/profilegen/src/resources/dir_metadata.json

```diff
@@ -0,0 +1,292 @@
+{
+  "Microsoft.AnalysisServices": {
+    "path": "../../../management/azure_mgmt_analysis_services/",
+    "namespace": "Azure::ARM::AnalysisServices",
+    "module_require": "azure_mgmt_analysis_services"
+  },
+  "Microsoft.Authorization": {
+    "path": "../../../management/azure_mgmt_authorization/",
```

Comment:
My personal other preference would be to not go relative-path route in the configuration file. Absolute would be explicit and better to understand as well.

Reply:
Will consider this for future enhancement

Reply:
👍

Reply:
yes, please add it to the list of stuff to reevaluate